### PR TITLE
Enhance build tools doc

### PR DESCRIPTION
**Pull request by ditsuke**
### Summary of Changes
Update `build/README.md` with better grammar.

### Testing Instructions
Read the build doc.

### Documentation Changes Required
None

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -1,66 +1,69 @@
...
-- Transform any SCSS file to the respective CSS file
+- To copy files from the `node-modules` folder to the `media` folder.
+- Do any transformations on the copied files.
+- Update the version numbers on the XML files of the TinyMCE and CodeMirror editors.
+- Copy files from the `build/media_source` folder to the `media` folder.
+- Transform any modern JS to both ES2017 and transpile it to ES5.
+- Transform any SCSS file to the respective CSS file.
 
-For some of these operations some conventions were established, to simplify and speed up the process.
+For some of these operations, some conventions were established to simplify and speed up the process.
```
Comment (jwaisner) [inline build/README.md]:
```suggestion
For some of these operations, conventions were established to simplify and speed up the process.
```

File: build/README.md [hunk 2]
```diff
@@ -1,66 +1,69 @@
...
-- do any transformations on the copied files
-- Update the version numbers on the xml files of the editors tinyMCE and Codemirror
-- Copy files from the `build/media_source` folder to the `media` folder
-- Transform any modern JS to both ES2017 and transpile it to ES5
-- Transform any SCSS file to the respective CSS file
+- To copy files from the `node-modules` folder to the `media` folder.
+- Do any transformations on the copied files.
+- Update the version numbers on the XML files of the TinyMCE and CodeMirror editors.
+- Copy files from the `build/media_source` folder to the `media` folder.
+- Transform any modern JS to both ES2017 and transpile it to ES5.
```
Comment (jwaisner) [inline build/README.md]:
```suggestion
- Transform any modern JS to ES2017 and transpile it to ES5.
```

**Review by jwaisner — CHANGES_REQUESTED** [review]
Just a few tweaks if you want to accept them.

**Comment by ditsuke** [comment]
> 
> 
> Just a few tweaks if you want to accept them.

Thanks, applied. :)

**Comment by jwaisner** [comment]
I have tested this item :white_check_mark: successfully on 75b7573da8dcc7e174903e6b1639f789b665dc08<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35736">issues.joomla.org/tracker/joomla-cms/35736</a>.</sub>

**Comment by RickR2H** [comment]
I have tested this item :white_check_mark: successfully on 75b7573da8dcc7e174903e6b1639f789b665dc08<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35736">issues.joomla.org/tracker/joomla-cms/35736</a>.</sub>

**Comment by RickR2H** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35736">issues.joomla.org/tracker/joomla-cms/35736</a>.</sub>

**Comment by HLeithner** [comment]
Thanks

**Pull request merged**